lib/samson/hooks.rb

```diff
@@ -25,6 +25,53 @@ def active?
       def require
         super @path
       end
+
+      def add_lib_path
+        engine.config.autoload_paths += Dir["#{engine.config.root}/lib/**/"]
+      end
+
+      def add_decorators
+        engine.config.after_initialize do
+          Dir.glob(engine.config.root.join('app/decorators/**/*_decorator.rb')).each do |c|
+            require_dependency(c)
+          end
+        end
+      end
+
+      def precompile_assets
```

Comment:
the method  is not really 'precompiling' the assets, so maybe something like `add_assets_to_precompile` would be better